### ISSUE TITLE
Weights.txt is empty and F1 score plot is blank

### ISSUE THREAD

**Issue by oluwayetty** [flairNLP/flair#1878]
Hi all, I just started using Flair some weeks ago. Thanks to the brilliant contributors.
For a weird reason that I am not able to investigate, the weights.txt and the F1 plot are blank after my model has successfully trained. I am building a NER model.

Here is a screenshot from the training. 
![image](https://user-images.githubusercontent.com/10073272/93849091-86d5ff00-fcab-11ea-9a19-e4ffc25053f8.png)

And a snapshot of the relevant part of my code.
![image](https://user-images.githubusercontent.com/10073272/93849371-4925a600-fcac-11ea-85e2-bd5cd9129d5b.png)

And a plot of the loss and f1 during training. You will observe F1 is blank.
![training](https://user-images.githubusercontent.com/10073272/93849497-90139b80-fcac-11ea-8b4b-cfd6f41295b7.png)

Is there anything I may be missing out for this bug to occur? Thanks

**Comment by alanakbik** [comment]
Hello @oluwayetty we made some changes that the `weights.txt` is no longer written out by default since for some models this file becomes huge (big transformer models). 

To write out weights, you need to set `write_weights=True` when training, e.g.: 

```python
trainer.train('ner_model',
              learning_rate=0.1,
              mini_batch_size=32,
              [....]
              write_weights=True,
              )
```

**Comment by oluwayetty** [comment]
Thank you for your prompt response @alanakbik 
For the F1 score, I traced the warning code of the screenshot below to this line: https://github.com/flairNLP/flair/blob/master/flair/visual/training_curves.py#L48 
![image](https://user-images.githubusercontent.com/10073272/93860547-89daea80-fcbf-11ea-85a7-bc125cd6a55b.png)

I also observed that my loss.tsv file doesn't contain TEST_F1 
![image](https://user-images.githubusercontent.com/10073272/93860791-e1795600-fcbf-11ea-9486-699f2223e21f.png)

Now, I am trying to understand why F1 score isn't in my tsv file. Is there a parameter that I am expected to add to my training to include the F1 score? Thanks

Below is my train code: 

```
trainer.train('ner-model',
              learning_rate=0.1,
              mini_batch_size=32,
              anneal_factor=0.5,
              write_weights=True,
              max_epochs=3,
              train_with_dev=False, # If True, training is performed using both train+dev data
              monitor_test=True, # If True, test data is evaluated at end of each epoch
              shuffle=True, # If True, data is shuffled during training
              embeddings_storage_mode='gpu')
```

**Comment by alanakbik** [comment]
Hello @oluwayetty the F1 score is only used for evaluation in sequence tagging models that use BIO or BIOES annotation in their tags. Otherwise, we use Accuracy as default. What dataset are you using and does it have span annotations?

**Comment by oluwayetty** [comment]
Oh I see, my data doesn't have BIO or BIOES annotation.
However, it does have span annotations? Is that a problem? Do you have a suggestion in regards to having span annotations?
Here is a sample of my data:

```
Transfer,O
for,O
USD,O-CURRENCY
to,O
Benson Limited,O-COMPANY
```
